app/models/correspondence_type.rb

```diff
@@ -100,39 +100,39 @@ def self.by_report_category
   end
 
   def self.foi
-    find_by_abbreviation! 'FOI'
+    find_by! abbreviation: 'FOI'
```

Comment:
Oh I think this is a false positive from rubocop. This is suppose to be calling a custom class method defined above `.find_by_abbreviation!` so probably we want to keep it as it is.